Capstone/Assets/Game/Scripts/AIDirector.cs

```diff
@@ -0,0 +1,78 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Bladesmiths.Capstone
+{
+    public class AIDirector : MonoBehaviour
+    {
+        public GameObject[] enemyPrefabs;
+        public List<Enemy> enemyGroup;
+        public LinkedList<Enemy> attackQueue;
+        private static AIDirector instance;
+        private bool enemyAttacking;
+
+
+        public static AIDirector Instance { get { return instance; } }
```

Comment:
This could be a little shorter if you want, just by using the arrow. So get => instance;